README.md

```diff
@@ -19,6 +19,7 @@ The currently supported functionality includes:
 - Deleting all ECS services in an AWS account
 - Deleting all EKS clusters in an AWS account
 - Deleting all RDS DB instances in an AWS account
+- Deleting all S3 buckets in an AWS account - except for buckets tagged with Key=cloud-nuke-excluded Value=true
```

Comment:
Seems like a `cloud-nuke-excluded` tag is a good general-purpose mechanism we should support globally across all resource types. Perhaps file an issue to track that and see if anyone wants to submit a PR to support it?

Reply:
Alternatively... Perhaps we could support an `--exclude-tag key=value` mechanism that excludes specific tags for all resource types that support tags? Either one makes sense to track in a GitHub issue.

Reply:
Filed https://github.com/gruntwork-io/cloud-nuke/issues/103